src/components/va-alert/va-alert.css

```diff
@@ -41,8 +41,9 @@ div.body {
   vertical-align: middle;
   width: 100%;
 }
-::slotted(:not([slot])){
+::slotted(:not([slot])) {
   margin-top: 2rem;
+  font-size: 1.6rem;
```

Comment:
Why do this instead of removing the `font-size: 2rem` in the `:host` selector?

Reply:
I'll see what that looks like :+1:

Reply:
By having the `font-size: 2rem` in the `:host` selector along with `font-size: 1em` in the `::slotted` selector we ensure that `h4` and other headers in the `headline` slot appear as 2rem regardless of what they are styled as in the light DOM.

Reply:
Ahh, I see. If that's what we're aiming for, I'd recommend styling for just that specifically.
```css
h1, h2, h3, h4, h5, h6 {
  font-size: 2rem
}
```